docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+version: '3.1'
+
```

Comment:
Can we move this file to `scripts/` if possible?  Would like to avoid cluttering the top-level dir if possible.

Reply:
We can; the main reason for keeping it on top-level is to have the docker-compose command easier (i.e. not needing to specify the yml file). It's only used to make local development easy, e.g. someone wanting to start a container locally.

Reply:
Convention / precedence with `piksi_buildroot` is for everything to go through the Makefile, so config files, environment variables, etc can all be set-up there.  I'm not opposed to changing this, but I'd want a more system-wide change that includes the other docker things here if we're going to do that.  For example, `make docker-setup` and `make docker-run` launches a local docker build environment.  Perhaps we could add `make docker-jenkins`?